flink-learning-common/src/test/java/com/zhisheng/common/utils/DateUtilTests.java

```diff
@@ -0,0 +1,87 @@
+package com.zhisheng.common.utils;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class DateUtilTests {
+
+    @Test
+    public void testFormat() {
+        Date date1 = new Date(1556665200000L);
+        Date date2 = new Date(1556723220000L);
+        Date date3 = new Date(1556723235000L);
+        Date date4 = new Date(1556723235000L);
+
+        Assert.assertEquals(date1.getTime(), DateUtil.format(date1));
+
+        Assert.assertEquals("2019-05-01", DateUtil.format(date1.getTime(), DateUtil.YYYY_MM_DD));
+        Assert.assertEquals("2019-05-01", DateUtil.format(date1, DateTimeZone.getDefault(), DateUtil.YYYY_MM_DD));
+
+        Assert.assertEquals(date1.getTime(), DateUtil.format("2019-05-01", DateUtil.YYYY_MM_DD));
+        Assert.assertEquals(date2.getTime(), DateUtil.format("2019-05-01 16:07", DateUtil.YYYY_MM_DD_HH_MM));
+        Assert.assertEquals(date3.getTime(), DateUtil.format("2019-05-01 16:07:15", DateUtil.YYYY_MM_DD_HH_MM_SS));
+        Assert.assertEquals(date4.getTime(), DateUtil.format("2019-05-01 16:07:15.0", DateUtil.YYYY_MM_DD_HH_MM_SS_0));
```

Comment:
Instead of creating `date2`, `date3`, `date4` using `long`s and then calling their `getTime()` methods (which returns same `long`), I would delete the instantiations (lines 18,19,20) and just put the `long`s directly into the assertions.

Reply:
Done